### DISCUSSION TITLE
User requests never show up in Radarr (only admin requests)

### DISCUSSION THREAD

**Discussion by jairajuku** [sct/overseerr#2070] [Help]
I'm super new to using Radarr and Overseerr, and Overseerr has been fantastic, except when I have users request things, they never make it over to Radarr. I tried the same thing, and they synced just fine. I'm not sure if there's something that can be manually triggered to update requests. The only difference I can think of is I'm requesting locally where users are requesting through a custom URL. Their requests still show up in Overseerr, but they just sit in requested.

**Comment by danshilm** [comment]
Do you have any errors in the Overseerr or Radarr logs when your users request content? If so, can you put the logs in a [gist](https://gist.github.com) and paste the link here?
